sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/TestUtils.java

```diff
@@ -241,5 +249,43 @@ private static boolean shouldServiceVersionBeTested(DocumentAnalysisServiceVersi
         return Arrays.stream(configuredServiceVersionList).anyMatch(configuredServiceVersion ->
             serviceVersion.getVersion().equals(configuredServiceVersion.trim()));
     }
+    public static FormRecognizerAudience getAudience(String endpoint) {
+        String authority = getAuthority(endpoint);
+        switch (authority) {
+            case AzureAuthorityHosts.AZURE_PUBLIC_CLOUD:
+                return FormRecognizerAudience.AZURE_RESOURCE_MANAGER_PUBLIC_CLOUD;
+
+            case AzureAuthorityHosts.AZURE_CHINA:
+                return FormRecognizerAudience.AZURE_RESOURCE_MANAGER_CHINA;
+
+            case AzureAuthorityHosts.AZURE_GOVERNMENT:
+                return FormRecognizerAudience.AZURE_RESOURCE_MANAGER_GOVERNMENT;
+
+            default:
+                return null;
+        }
+    }
+
+    public static String getAuthority(String endpoint) {
+        if (endpoint == null) {
```

Comment:
Food for thought: if this code is ever reused for prod/user-facing scenarios, it might be better in the long run to actually parse the endpoint into a URL data structure and check the domain name is a subdomain of a known Azure domain rather than just contains '.io' or '.cn'. This is probably fine for testing.